rest-api-spec/src/main/resources/rest-api-spec/test/indices.get/10_basic.yml

```diff
@@ -160,3 +160,11 @@ setup:
   - is_true: test_index_2.settings
   - is_true: test_index_3.settings
 
+---
+"Should return an exception when querying invalid indices":
```

Comment:
This test needs to assert on the HTTP status.

Reply:
Oh I was unaware we could do that. I must've missed it in the docs!

Reply:
You might have to add a specific catch for `bad_request` (400).

Reply:
#26539

Reply:
ill block this on it being merged